src/NuGet.Clients/NuGet.CommandLine/Common/NuGetTestMode.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Common
+{
+    // Note: we may delete this logic once we no longer depend on NuGet.Core's PackageServer type,
+    // as it is embedded in NuGet.Protocol.Core.Types.UserAgent.
+    internal static class NuGetTestMode
+    {
+        private const string _testModeEnvironmentVariableName = "NuGetTestModeEnabled";
```

Comment:
When setting an environment variable in PowerShell using `$env:VariableName = value`, it is a process-level environment variable, and only visible to, and lasts only as long as, your current PowerShell session. (source: https://technet.microsoft.com/en-us/library/ff730964.aspx)